### DISCUSSION TITLE
Problem with installation

### DISCUSSION THREAD

**Discussion by Leandro-Bandeira** [ctu-mrs/mrs_uav_system#123] [General]
mavlink.sh: "make > /dev/null" command failed with exit code 2

mavros.sh: "$MY_PATH/mavlink.sh --install $DRYRUN" command failed with exit code 2

install.sh: "bash $MY_PATH/dependencies/mavros.sh --compile" command failed with exit code 2

**Comment by matemat13** [comment]
Hello, did you use the following command to install the system (as per the installation instructions here: https://github.com/ctu-mrs/mrs_uav_system#installation)?
```
cd /tmp
echo '
GIT_PATH=~/git
mkdir -p $GIT_PATH
cd $GIT_PATH
sudo apt-get -y install git
git clone https://github.com/ctu-mrs/mrs_uav_system
cd mrs_uav_system
git checkout master
git pull
./install.sh -g $GIT_PATH
source ~/.bashrc' > clone.sh && source clone.sh
```

**Reply by Leandro-Bandeira** [reply]
Yeah, I used this command and followed every step

**Reply by matemat13** [reply]
On which operating system are you attempting to install the system?

**Reply by Leandro-Bandeira** [reply]
Ubuntu 20.04 LTS

**Reply by Leandro-Bandeira** [reply]
Ubuntu 20.04

**Reply by matemat13** [reply]
And when do these errors appear?
